src/main/scala/cromwell/binding/values/WdlValueJsonFormatter.scala

```diff
@@ -11,7 +11,11 @@ object WdlValueJsonFormatter extends DefaultJsonProtocol {
       case f: WdlFloat => JsNumber(f.value)
       case b: WdlBoolean => JsBoolean(b.value)
       case f: WdlFile => JsString(f.value)
-      case o: WdlObject => JsObject()
+      case o: WdlObject => new JsObject(
```

Comment:
Not really, I was thinking when getting the output you would want to know if the result you get is a `WdlMap` or a `WdlObject` but maybe it doesn't matter actually.